keyboards/flx/virgo/keymaps/default/keymap.c

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 MechMerlin
+/* Copyright 2020 Flexerm
```

Comment:
lol, i'd appreciate if you just add your name rather than removing mine.

Reply:
None of your original code remains in the keymap.c file, the layout and keymap has changed. I removed your name out of curtesy since I am an amateur at this. 

Note it’s been kept anywhere your code remains.

Reply:
Alright!